src/main/java/org/opensearch/sdk/SDKClient.java

```diff
@@ -597,5 +601,7 @@ public Cancellable rolloverIndex(RolloverRequest rolloverRequest, ActionListener
         public Cancellable getAliases(GetAliasesRequest getAliasesRequest, ActionListener<GetAliasesResponse> listener) {
             return this.indicesClient.getAliasAsync(getAliasesRequest, RequestOptions.DEFAULT, listener);
```

Comment:
Shouldn't we add `options` for all the requests then? Why only `search`?

Reply:
> Shouldn't we add `options` for all the requests then? Why only `search`?

\<insert facepalm emoji\>

Yeah, it should replace the default everywhere the default is used.  This is what happens when you look at the whole sausage-making process and miss things at the end.